vignettes/datatable-reference-semantics.Rmd

```diff
@@ -92,7 +92,7 @@ It can be used in `j` in two ways:
     # when you have only one column to assign to you
     # can drop the quotes and list(), for convenience
     DT[, colA := valA]
-	  ```
+    ```
```

Comment:
@MichaelChirico I discovered a really weird thing here and I wonder if you could help me understand it. The fix you made here was correct, but if you download the source package from CRAN: https://cran.r-project.org/src/contrib/data.table_1.14.8.tar.gz you will find that the first two spaces are actually a tab character `\t`.